logging/logger.go

```diff
@@ -51,6 +52,28 @@ var (
 	logRotateBytes    int
 )
 
+// To check if the provided log file path is valid or not
+func validateLogFilePath(logFileDir string, ui cli.Ui) bool {
+	// dummy file for validation directory existence and write permission
+	dummyFile := filepath.Join(logFileDir, "dummy.log")
+
+	file, err := os.Create(dummyFile)
+	defer os.Remove(dummyFile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			ui.Error(fmt.Sprintf("Cannot access directory: %v : no such file or directory", logFileDir))
```

Comment:
OPT: improve a bit error message: "Logs directory %v  does not exist, please create it"